docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -121,70 +128,64 @@ export default function Page() {
 
-#### Conditions for Soft Navigation
+Learn more about how the [Router Cache](/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#caching-data) works and how to configure it.
 
-On navigation, Next.js will use soft navigation if the route you are navigating to has been [**prefetched**](#prefetching), and either doesn't include [dynamic segments](/docs/app/building-your-application/routing/dynamic-routes) **or** has the same dynamic parameters as the current route.
+#### 3. Partial Rendering
 
-For example, consider the following route that includes a dynamic `[team]` segment: `/dashboard/[team]/*`. The cached segments below `/dashboard/[team]/*` will only be invalidated when the `[team]` parameter changes.
+Partial rendering means only the route segments that change on navigation re-render on the client, and any shared segments are preserved.
 
-- Navigating from `/dashboard/team-red/*` to `/dashboard/team-red/*` will be a soft navigation.
-- Navigating from `/dashboard/team-red/*` to `/dashboard/team-blue/*` will be a hard navigation.
+For example, when navigating between two sibling routes, `/dashboard/settings` and `/dashboard/analytics`, the `settings` and `analytics` pages will be rendered, and the shared `dashboard` layout will be preseved.
 
-### Hard Navigation
+<Image
+  alt="How partial rendering works"
+  srcLight="/docs/light/partial-rendering.png"
+  srcDark="/docs/dark/partial-rendering.png"
+  width="1600"
+  height="945"
+/>
 
-On navigation, the cache is invalidated and the server refetches data and re-renders the changed segments.
+Without partial rendering, each navigation would cause the full page to re-render on the server. Rendering only the segment that changes reduces the amount of data transferred and execution time, leading to improved performance.
 
-### Back/Forward Navigation
+#### 4. Soft Navigation
 
-Back and forward navigation ([popstate event](https://developer.mozilla.org/en-US/docs/Web/API/Window/popstate_event)) has a soft navigation behavior. This means, the client-side cache is re-used and navigation is near-instant.
+By default, the browser performs a hard navigation between pages. This means the browser reloads the page and resets the React state. However, in Next.js, the App Router uses soft navigation. This means React renders the segments that have changed while preserving browser and React state, and there is no page reload.
 
-### Focus and Scroll Management
+#### 5. Back and Forward Navigation
 
-By default, Next.js will set focus and scroll into view the segment that's changed on navigation.
+By default, Next.js will maintain the scroll position for backwards and forwards navigation, and re-use route segments in the [Router Cache](/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#caching-data).
```

Comment:
How does this fit in with `experimental: {
    scrollRestoration: true,
  },`? Is that functionality build in to the app router?

Reply:
I'll defer this one to @timneutkens as I'm not too familiar with the implementation, afaik `scrollRestoration` is `pages` router only.